td.server/src/env/Google.js

```diff
@@ -0,0 +1,25 @@
+import { Env } from './Env.js';
+
+class GoogleEnv extends Env {
+    constructor () {
+        super('Google');
+    }
+
+    get prefix () {
+        return 'GOOGLE_';
+    }
+
+    // Note that the actual env var will be prepended with GOOGLE_
+    get properties () {
+        return [
+            { key: 'CLIENT_ID', required: false },
+            { key: 'CLIENT_SECRET', required: false },
+            { key: 'SCOPE', required: false, defaultValue: 'openid email profile' },
+            { key: 'REDIRECT_URI', required: false, defaultValue: 'http://localhost:8080/oauth-return' },
```

Comment:
Out of curiosity, why is the port here different than in example.env?

Reply:
Oh, I think I have write the sample redirect url from the Google Drive API doc.
So is it ok if I update it with http://localhost:3000/api/oauth/return ?

Reply:
Sure!

Reply:
Hi @ericfitz .
Just updated the default env variable for Google Redirect URL. Also this PR has passed all the checks, Please check.

Reply:
Hello @ericfitz .

Also, we can end the current contract in Upwork and leave reviews for each other, It'd be very helpful for me.
As you can see, my account is new so it'd better if I have more good reviews from clients.
There is two kind of score in Upwork when you end the contract, Please give me private score 10/10, and public score 5/5 and your reviews for my work.
If you need my help again, we can start a new contract there, in this way I can have more reviews, Thank you very much.